folder_paths.py

```diff
@@ -42,41 +43,41 @@
 input_directory = os.path.join(os.path.dirname(os.path.realpath(__file__)), "input")
 user_directory = os.path.join(os.path.dirname(os.path.realpath(__file__)), "user")
 
-filename_list_cache = {}
+filename_list_cache: Dict[str, Tuple[List[str], Dict[str, float], float]] = {}
```

Comment:
Minor: we can `from __future__ import annotations` to use the Python 3.10 syntax here, i.e. `dict[str, tuple[list[str], dict[str, float], float]]`

Reply:
Thank you for your feedback. I implemented your suggestion.